addons/mail/static/src/model/model_field.js

```diff
@@ -151,6 +152,11 @@ class ModelField {
          * and 'many2many'.
          */
         this.relationType = relationType;
+        /**
+         * Mark the field as required. Empty value is only undefined. Null,
```

Comment:
_"_
_Determine whether the field is required or not._

_Empty value is systematically `undefined`._
_`null` or empty string are NOT considered empty value, meaning these values meet the requirement._
_"_